packages/xo-web/src/xo-app/settings/logs/index.js

```diff
@@ -55,15 +73,7 @@ const COLUMNS = [
         <pre className={styles.widthLimit}>
           {log.data.error && log.data.error.message}
         </pre>
-        {log.data.error && log.data.error.code === 'LICENCE_RESTRICTION' && (
-          <a
-            href='https://xcp-ng.org/'
-            rel='noopener noreferrer'
-            target='_blank'
-          >
-            {_('logSuggestXcpNg')}
-          </a>
-        )}
+        <LogInfo log={log} />
```

Comment:
I think, for `XapiError` is better to have like this  `XenServer/XCP-ng error:  error`. 
What do you think?

Reply:
There will probably not be enough room for `'XenServer/XCP-ng error:'` and the error message, though.